app/javascript/packs/ping_things/user_stats.js

```diff
@@ -0,0 +1,25 @@
+import Vue from 'vue/dist/vue.esm'
+import UserStats from './user_stats.vue'
+import TurbolinksAdapter from 'vue-turbolinks';
+import axios from 'axios'
+import ActionCableVue from "actioncable-vue";
+
+Vue.use(TurbolinksAdapter);
+
+axios.defaults.headers.get["Authorization"] = window.api_authorization
+
+Vue.use(ActionCableVue, {
+  debug: true,
+  debugLevel: "error",
+  connectionUrl: "/cable",
+  connectImmediately: true,
+});
+
+document.addEventListener('turbolinks:load', () => {
+  const pt_stats = new Vue({
+    el: '#ping-thing-stats',
```

Comment:
this id is already used for stats bar on homepage
also we don't plan to display user stats anywhere else so it looks like we don't need this file